### DISCUSSION TITLE
version 2.4.11

### DISCUSSION THREAD

**Discussion by gekigek99** [gekigek99/minecraft-vanilla-server-hibernation#195] [Releases]
### major updates
- added msh console input reserved line
- added query support

### minor updates
- added parser for server.properties
- added query challenge library
- added query stats from minecraft server when it's warm
- added check for `list` command output
- added check while loading version/protocol
- added instructions to prevent ms watchdog crash in readme
- improved msh log trace layout

### bugfix
- bugfix machine mem value overflow on 32bit arch
- bugfix ms info first time request
- bugfix list unexpected output
- bugfix log graphics
- bugfix ms command space parsing
- bugfix challenge list element removal
- bugfix 6 digit number challenge generation

**Full Changelog**: https://github.com/gekware/minecraft-server-hibernation/compare/v2.4.10...v2.4.11

<hr /><em>This discussion was created from the release <a href='https://github.com/gekware/minecraft-server-hibernation/releases/tag/v2.4.11'>version 2.4.11</a>.</em>

**Comment by Br31zh** [comment]
Hello!

It had been a while since I had stop playing and maintaining my Minecraft server, and then neglected this project. I just updated it to 2.4.11 by updating my server to 1.19.3, everything seems to be working fine (including the bug where I never got back to you, sorry for that). Thank you again for your work!
